Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/Solver.cs

```diff
@@ -39,39 +46,41 @@ public abstract class Solver : MonoBehaviour
         /// <summary>
         /// The handler reference for this solver that's attached to this <see cref="GameObject"/>
         /// </summary>
+        [SerializeField]
+        [HideInInspector]
```

Comment:
With a custom inspector, so we still need this?

Reply:
Where's the custom inspector?